assets/src/components/v2/departures/route_pill.tsx

```diff
@@ -54,6 +54,8 @@ const TextRoutePill: ComponentType<TextPill & { outline?: boolean }> = ({
 
   if (size) {
     modifiers.push(size);
+  } else if (text.startsWith("TR")) {
+    modifiers.push(text.length > 3 ? "small" : "large");
```

Comment:
A note - switched the logic here because `Number(text)` was returning NaN for all `TR` text. I can modify the logic there, but I'm not sure what other text can show up within this flow that might actually want to be NaN.